src/lib/markbind/src/parser.js

```diff
@@ -449,14 +274,6 @@ class Parser {
     context.cwf = config.cwf || file; // current working file
 
     return new Promise((resolve, reject) => {
-      let actualFilePath = file;
```

Comment:
removed this as 
- confirmed that for expressive layouts, the file will always exist ( ie. the layout file ) ( there is also a explicit check to abort in `generateExpressiveLayout` if the file does not exist )
- it adds confusion to the changes below, since `actualFilePath` will always `== file`